app/scripts/lib/state-corruption-errors.ts

```diff
@@ -0,0 +1,106 @@
+import browser from 'webextension-polyfill';
+import log from 'loglevel';
+import { escape as lodashEscape } from 'lodash';
```

Comment:
I accidentally noticed that if a string has a `"` in it (or even `', `<`, or `&`) it can break some of this hand-stitched HTML. React handles it automatically, but we were doing that here.

So I'm now `escape`ing usages of our locale strings in this file just to be safe. It doesn't _have_ to be in this PR, so let me know if you want me to remove it.